src/amplitude-client.js

```diff
@@ -168,7 +168,7 @@ AmplitudeClient.prototype.init = function init(apiKey, opt_userId, opt_config, o
     if (AsyncStorage) {
       Promise.all([
           AsyncStorage.getItem(this._storageSuffix),
-          AsyncStorage.getItem(this.options.unsentKey),
+          AsyncStorage.getItem(this.options.unsentKey + this._storageSuffix),
           AsyncStorage.getItem(this.options.unsentIdentifyKey),
```

Comment:
you probably want to fix this in all places? like here

Reply:
you potentially need a migration as well, if there are any lingering events under the old keys, to migrate them to the new ones?